src/Core/CommandContext.ts

```diff
@@ -9,9 +9,7 @@ export class CommandContext {
     public constructor(message: Message, prefix: string) {
         const messageContent = message.content.toLowerCase();
         const args = messageContent.split(' ');
```

Comment:
@qcarpentier Please, give me the types for this or add a commit on this pull request #13

Reply:
**Fixed, see commit fec4d04**